cmd/lakectl/cmd/root.go

```diff
@@ -527,8 +535,8 @@ func initConfig() {
 
 	// Inform viper of all expected fields.
 	// Otherwise, it fails to deserialize from the environment.
-	var cfg Configuration
-	keys := lakefsconfig.GetStructKeys(reflect.TypeOf(cfg), "mapstructure", "squash")
+	var conf Configuration
+	keys := lakefsconfig.GetStructKeys(reflect.TypeOf(conf), "mapstructure", "squash")
```

Comment:
It's `cfg` everywhere, why does this one deserve to be the `conf`?

Reply:
Because this is a local variable that shadows a global one and that is a nono